src/Approximations/decompositions.jl

```diff
@@ -255,6 +256,40 @@ function decompose(S::LazySet, block_options; block_size::Int=1)
     return decompose(S, partition, block_options)
 end
 
+"""
+    project(S::LazySet{N},
+            block::AbstractVector{Int},
+            ::Nothing,
+            [n]::Int=dim(S)
+           ) where {N<:Real}
+
+Project a high-dimensional set to a given block by using a concrete linear map.
+
+### Input
+
+- `S`       -- set
+- `block`   -- block structure - a vector with the dimensions of interest
+- `nothing` -- used for dispatch
+- `n`       -- (optional, default: `dim(S)`) ambient dimension of the set `S`
+
+### Output
+
+A set representing the projection of the set `S` to block `block`.
+
+### Algorithm
+
+We apply the function `linear_map`.
+"""
+@inline function project(S::LazySet{N},
```

Comment:
I'm not sure about inlining.. is there some general rule that applies to using `@inline`?

Reply:
I copied this from the other `project` methods.

Reply:
Ok, thanks.